.travis.yml

```diff
@@ -11,3 +11,7 @@ branches:
   only:
     - master
     - 1.x_dev
+
+script:
+  - xcodebuild -workspace push-sdk.xcworkspace -scheme push-sdk -sdk iphonesimulator build
+  - xcodebuild -workspace push-sdk.xcworkspace -scheme push-sdk -destination 'platform=iOS Simulator,name=iPhone 7,OS=10.0' test
```

Comment:
@damienomurchu when there are no scripts I think it just try to build the project.
As we have tests and want travis to run them, then we have to add this kind of script entries to the travis file.